extensions/amp-form/0.1/amp-form.js

```diff
@@ -769,12 +773,19 @@ export class AmpForm {
   }
 
   /**
-   * Transition the form to the submit success state.
+   * Transition the form to the submit-success or submit-error state depending on the response status.
    * @param {!JsonObject} response
    * @return {!Promise}
    * @private
    */
-  handleSsrTemplateSuccess_(response) {
+  handleSsrTemplateResponse_(response) {
+    if (
+      response.init &&
```

Comment:
`JsonObject` type should only be dereferenced via bracket notation to avoid minification issues.